app/grants/views.py

```diff
@@ -2613,13 +2612,13 @@ def contribute_to_grants_v1(request):
             })
             continue
 
-        tx_id = contribution.get('tx_id', None)
-        if not tx_id:
-            invalid_contributions.append({
-                'grant_id': grant_id,
-                'message': 'error: tx_id is mandatory param'
-            })
-            continue
+        # tx_id = contribution.get('tx_id', None)
```

Comment:
made this not mandatory as other cross chains might not be able to provide tx_id , commented out in case if want to keep this for some reason

Reply:
Good call - do you want to add a quick comment in the code explaining it?

Reply:
will add it in a follow up commit 🙌